bbk-requirements.md

```diff
@@ -0,0 +1,76 @@
+# Secure Payment Confirmation: Browser Based Key Requirements and Design Considerations
+
+Status: This is a **draft** document without consensus.
+
+Though this document we seek to build consensus around requirements and design considerations for adding a browser-based key pair (BBK) to the results of Secure Payment Confirmation to serve as a possession factor during authentication. For discussion, see [issue 271](https://github.com/w3c/secure-payment-confirmation/issues/271).
+
+## Motivation
+
+Secure Payment Confirmation provides a convenient "sign what you see" experience for a user to agree to the terms and conditions of a transaction, and where Web Authentication is used to generate cryptographic evidence of the user's agreement.
+
+The payments industry has indicated that SPC would further benefit from a device binding capability. As WebAuthn passkeys can now be synced, it can be argued that they no longer meet strict 2FA requirements (being no longer a signal of device possession), and so SPC (like WebAuthn) is reduced to a single factor (biometric or possession, depending on the authentication method used).
+
+The Web Authentication Working Group has considered but not adopted device binding proposals (cf. [SPK](https://github.com/w3c/webauthn/pull/1957) and [DPK](https://github.com/w3c/webauthn/issues/1658)).
+
+Therefore, the Web Payments Working Group plans to add a "browser-based key (BBK)" SPC. This document endeavors to capture requirements for the BBK functionality.
+
+## Assumptions
+
+* A relying party will perform some ID &amp; V process before trusting a (new) BBK. That ID &amp; V process might take place before a Web Authentication registration (and thus, if the BBK is returned as part of the Web Authentication registration, the RP would not likely step up the user a second time). In the case of a synched passkey, when the RP first sees a BBK on a new user device, in the absence of other trust signals, the RP would likely perform some ID &amp; V process in order to trust the new BBK, and we consider that an acceptable user experience on a new device.
+
+## Requirements
+
+***Notes:***
+
+* These requirements are not prioritized.
+* "MUST", "SHOULD", and "MAY" are used per [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+
+### Association with passkeys
+
+* A given passkey should have only one associated BBK at any given time for a given user agent user profile.
+
+* A BBK must only ever be associated with one passkey.
+
+* If a BBK is deleted, the user agent should generate a new BBK associated with the same passkey.
+
+* At authentication time, the Web Authentication challenge will be signed by the passkey and the associated BBK.
```

Comment:
`Web Authentication challenge` --> `client data` (because the signature is over all of client data, which includes the Web Authentication challenge, among other things).